install_fzy_native_sorter

```diff
@@ -0,0 +1,3 @@
+git submodule init
+git submodule update
+cd external_modules/fzy_lua_native/ && make
```

Comment:
IIRC most popular plugin managers already clone a repo with `--recurse-submodules` so this shouldn't be needed (unless installing manually).

And the make command could be done like:

```
make -C external_modules/fzy_lua_native
```

Reply:
I just noticed that, thanks! I updated the installation file and readme for the manual installation thing.